test/integration/kind_cluster_test.go

```diff
@@ -0,0 +1,42 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/kind"
+)
+
+func TestKongProxyClusterWithMetalLB(t *testing.T) {
+	config := kind.ClusterConfigurationWithKongProxy{
+		DockerNetwork: kind.DefaultKindDockerNetwork,
+		EnableMetalLB: true,
+	}
+
+	cluster, ready, err := config.Deploy(context.Background())
+	defer cluster.Cleanup()
```

Comment:
shall we `require.NoError` on `Cleanup()`?

Reply:
Yes :+1: